.github/workflows/build.yml

```diff
@@ -48,7 +48,11 @@ jobs:
       matrix:
         package: ${{ fromJson(needs.find-all-packages.outputs.packages) }}
         config:
-          - { os: ubuntu-22.04, compiler: clang-15 }
+          - { os: ubuntu-22.04, compiler: clang-15, host-profile: host }
+          - { os: ubuntu-22.04, compiler: ndk-26.3.11579264, host-profile: armv8 }
+          - { os: ubuntu-22.04, compiler: ndk-26.3.11579264, host-profile: x86_64 }
+          - { os: ubuntu-22.04, compiler: ndk-26.3.11579264, host-profile: armv7 }
+          - { os: ubuntu-22.04, compiler: ndk-26.3.11579264, host-profile: x86 }
```

Comment:
I am not sure yet if Android makes sense here but I would say at maximum one? Because otherwise we launch a very big amount of jobs.

I think `compiler` should stick with the `clang` rather than `ndk-26.3.11579264`. I guess we could have build OS and host OS to make this a bit cleaner?

Reply:
Well it depends on how we want to use this odr-index repo.

Small matrix here means faster builds here and our artifactory is more like a recipe index, but this means slow builds for downstream users, because most of the time, downstream users would have to build these conan packages from source.

Big matrix here means slower builds here, but fast builds for downstream users, because downstream users would be able to find binary packages that were built here. Mind you, downstream users (odr.droid) need binaries from all 4 Android ABIs.

How is artifactory charging us for the repository? Does the amount of binaries that we upload have an actual cost? Or is it in gigabytes and we don't actually have to care? Big matrix here does not have that much of an impact on GitHub side, the repo is open source, so we don't have to worry about GitHub charges, only the run time, but if all the dependencies are already available as binaries, each of them wouldn't take that long, it's not in the hours as it was when I was building pdf2htmlEX dependencies from source each time :D

As for the `compiler`, `clang` and `ndk-26.3.11579264`, I don't really care about the actual naming, what I need is a way to specify build and host profiles. And I also need some kind of a variable to specify NDK version, because NDK needs to be installed manually